<launch>
	<!-- run the position_control.cpp -->

	<node pkg="px4_command" type="autonomous_landing" name="autonomous_landing" output="screen">

		<!-- load blacklist, config -->
                <rosparam command="load" file="$(find px4_command)/config/autonomous_landing.yaml" />
	</node>
</launch>
